en/components/radialgauge.md

```diff
@@ -0,0 +1,97 @@
+
+### Demo
+
+<div class="sample-container" style="height: 550px">
+    <iframe id="radial-gauge-sample-iframe" src='{environment:demosBaseUrl}/radial-gauge-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="radial-gauge-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+Create a scale by supplying `minimumValue` and `maximumValue` values and a needle can point to a value by setting the `value` property. The gauge also supports ranges, which provide visual cues for the scale.
+
+The gauge comes with a backing shape drawn behind the scale that acts as a background for the gauge. The backing can be circular, which turns the shape into a full circle, or it can be fitted, which makes it sweep the scale – only providing backing behind the scale portion of the gauge.
+
+Gauge needles are available in one of the several predefined shapes. The needle can have a pivot shape, which is placed in the center of the gauge. The pivot shape also takes one of the predefined shapes. Pivot shapes that include an overlay or an underlay can have a separate pivot brush applied to the shape. 
+
+This gauge supports only one scale, one needle, one set of tick marks and one set of labels. Practically all the changes made to the radial gauge control are animation enhancements. This animation is highly customizable by setting the `transitionDuration` property. You can achieve numerous needle shapes by fine tuning its shape parameters.
+
+### Dependencies
+The radial gauge exported as an `NgModule`, you need to import the _IgxRadialGaugeModule_ inside your `AppModule`:
+
+```typescript
+// app.module.ts
+import { IgxRadialGaugeModule } from '@infragistics/ignite-ui-xplatform-for-angular/Debug/ES5/igx-radialgauge-module';
+
+@NgModule({
+    imports: [
+        ...
+        IgxRadialGaugeModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+
+<div class="divider--half"></div>
+
+### Usage
+
+The following code demonstrates how create a radial gauge containing a needle and three comparative ranges on the scale.
+
+```html
+ <igx-radial-gauge value = "50"
+                   scaleStartExtent = "0"
+                   scaleEndExtent = "100">
+    <igx-radial-gauge.ranges>
+        <igx-radial-graph-range startValue="0"
+                                endValue="30"
+                                brush="red"/>
+        <igx-radial-graph-range startValue="30"
+                                endValue="60"
+                                brush="yellow"/>
+        <igx-radial-graph-range startValue="60"
+                                endValue="100"
+                                brush="green"/>
+    </igx-radial-gauge.ranges>         
```

Comment:
please remove `</igx-radial-gauge.ranges>` closing tag because it will throw errors! It is needed only in XAML